estate/models/estate_property.py

```diff
@@ -149,3 +151,16 @@ def action_cancel(self):
         )
     ]
 
+    _sql_constraints = [
+        (
+            "check_expected_price",
+            "CHECK(expected_price > 0)",
+            "Expected price must be strictly positive",
+        ),
+        (
+            "check_selling_price",
+            "CHECK(selling_price > 0)",
+            "Expected price must be strictly positive",
+        )
+    ]
+
```

Comment:
I think this is old code, this already has been merged

check this [line](https://github.com/Cindypsd/Odoo_Training/blob/main/estate/models/estate_property.py#L139)